lib/features/departments_tab/widgets/department_card.dart

```diff
@@ -9,11 +9,13 @@ import '../../../widgets/wide_tile_card.dart';
 
 class DepartmentCard extends StatelessWidget {
   final Department department;
-  const DepartmentCard(this.department, {super.key});
+  const DepartmentCard(this.department, {super.key, this.onClick});
+  final void Function()? onClick;
```

Comment:
maybe `VoidCallback` - not forcing though?

Reply:
I didn't know about it! Nice and clean actually